src/modules/common/components/SearchInput.js

```diff
@@ -1,33 +1,34 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { translate } from 'react-i18next'
 import { SearchIcon, Spacer, TextInput, Wrapper } from './SearchInput.styles'
 
 export class SearchInput extends React.Component {
   static propTypes = {
+    placeholderText: PropTypes.string.isRequired,
     filterText: PropTypes.string.isRequired,
     onFilterTextChange: PropTypes.func.isRequired,
     spaceSearch: PropTypes.bool,
-    onClickInput: PropTypes.func,
-    t: PropTypes.func.isRequired
+    onClickInput: PropTypes.func
   }
 
   onFilterTextChange = event => this.props.onFilterTextChange(event.target.value)
 
   render () {
+    const {onClickInput, filterText, placeholderText} = this.props
     return (
       <Spacer space={this.props.spaceSearch}>
         <Wrapper>
           <SearchIcon />
-          <TextInput placeholder={this.props.t('search')}
-                     defaultValue={this.props.filterText}
+          <TextInput placeholder={placeholderText}
+                     label={placeholderText}
```

Comment:
Is label really used? https://github.com/Integreat/integreat-webapp/blob/f67040565993824bbad672d52e49f9cc5b67358a/src/modules/common/components/SearchInput.styles.js#L10

Also a `<input>` does not have a label attribute: https://www.w3schools.com/tags/tag_input.asp

But there is an aria-label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

Reply:
what do you mean by "really used"? Changed to aria-label.